src/dev-mock-service/dev-mock-service.ts

```diff
@@ -0,0 +1,66 @@
+import type { RedditAPIClient, RedisClient } from "@devvit/public-api";
+import type { Devvit } from "@devvit/public-api";
+import type { HandlerOverride } from "./types/index.js";
+
+import {
+  createDevvRedis,
+  isRedisHandler,
+  redisHandler,
+} from "./redis-mock-service/index.js";
+import {
+  createDevvRedditApi,
+  isRedditApiHandler,
+  redditApiHandler,
+} from "./reddit-api-mock-service/index.js";
+import {
+  createDevvFetch,
+  httpHandler,
+  httpResponse,
+  isHttpApiHandler,
+} from "./http-mock-service/index.js";
+
+export enum DevMockMode {
+  Prod = "Prod",
+  Dev = "Dev",
+}
+
+export type DevMockService = {
+  devvRedis: RedisClient;
+  devvRedditApi: RedditAPIClient;
+  devvFetch: typeof fetch;
```

Comment:
suggest omitting the name devv prefix names (I make a comment about this later that it's a new term) because I don't think it's adding much useful context here. the user only needs to know which service is which and the typing covers that.

Reply:
This is done to avoid confusion with the actual instances of `redis` and `reddit`
Imagine the following scenario
```typescript
const {redis, reddit} = context;
const { devvRedis, devvRedditApi, devvFetch } = DevMock.createService({...
```

Without the prefix there is a potential name collision